package.json

```
@@ -107,26 +118,23 @@
     "test:storefront:lib": "nx test storefrontlib --source-map --code-coverage",
     "test:opf:lib": "nx test opf --source-map --code-coverage",
-    "dev:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 nx run storefrontapp:serve-ssr",
-    "serve:ssr": "node dist/storefrontapp-server/main.js",
-    "serve:ssr:ci": "NODE_TLS_REJECT_UNAUTHORIZED=0 SSR_TIMEOUT=0 node dist/storefrontapp-server/main.js",
-    "serve:ssr:dev": "cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 node dist/storefrontapp-server/main.js",
+    "serve:ssr": "node dist/storefrontapp/server/server.mjs",
+    "serve:ssr:watch": "node --watch-path=dist/storefrontapp/server/ dist/storefrontapp/server/server.mjs",
+    "serve:ssr:ci": "NODE_TLS_REJECT_UNAUTHORIZED=0 SSR_TIMEOUT=0 node dist/storefrontapp/server/server.mjs",
+    "serve:ssr:dev": "cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 node dist/storefrontapp/server/server.mjs",
     "test:ssr": "env-cmd -e dev nx test ssr-tests",
     "test:ssr:ci": "env-cmd -e ci nx test ssr-tests",
-    "prerender": "nx run storefrontapp:prerender --routes-file projects/storefrontapp/prerender.txt",
-    "prerender:dev": "env-cmd --no-override -e dev,$SPA_ENV cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 nx run storefrontapp:prerender --routes-file projects/storefrontapp/prerender.txt",
-    "prerender:cds": "env-cmd --no-override -e cds nx run storefrontapp:prerender --routes-file projects/storefrontapp/prerender.txt",
+    "prerender": "env-cmd --no-override -e dev,$SPA_ENV cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 nx run storefrontapp:prerender-production",
     "publish:docs": "npx --yes gh-pages@^2.2.0 -d documentation -m \"Compodocs updates\"",
     "test:cds:lib": "nx test cds --code-coverage",
-    "build:cds:ssr": "env-cmd --no-override -e dev,b2c,cds nx run storefrontapp:server:production",
-    "dev:cds:ssr": "env-cmd --no-override -e dev,b2c,cds cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 nx run storefrontapp:serve-ssr",
-    "prerender:cds:dev": "env-cmd --no-override -e dev,cds cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 nx run storefrontapp:prerender --routes-file projects/storefrontapp/prerender.txt"
+    "build:cds:ssr": "env-cmd --no-override -e dev,b2c,cds nx run storefrontapp:build:production",
+    "watch": "npm run build -- --watch"
```

[MEDIUM]
The watch build IMHO should use dev mode, so let's change it to 
```suggestion
    "watch": "npm run build -- --watch --configuration production"
```

(and btw. we might want to move it e.g. right under the "build" script, so they are close to each other in the package.json file)